### ISSUE TITLE
Cron monitors not working

### ISSUE THREAD

**Issue by gregory-claeyssens** [getsentry/sentry-laravel#805]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

4.0.0

### Steps to reproduce

Following the documentation to the letter by adding `->sentryMonitor()`
Example:

```
$schedule->command('cache:prune-stale-tags')->hourly()->sentryMonitor();
```

### Expected result

It simply work

### Actual result

An error is logged in Sentry
`Method Illuminate\Console\Scheduling\Event::sentryMonitor does not exist.`

**Comment by cleptric** [comment]
We need a bit more info, please, like the Laravel version. Is anything custom in terms of SDK setup? Anything else you can think of?

**Comment by rzv-me** [comment]
I got the same error. 

We are using Laravel Vapor.
The problem, from what I understand from the code, is that Laravel Vapor instantiates the console kernel
`$app->make(ConsoleKernelContract::class)->call('config:cache');` in `vendor/laravel/vapor-core/stubs/fpmRuntime.php`, but the `isApplicable()` function on `ConsoleIntegration`, returns true only when running in console.

My solution right now is to add 
```
    protected function addSentryMonitorMacro()
    {
        if(!SchedulingEvent::hasMacro('sentryMonitor')) {
            SchedulingEvent::macro('sentryMonitor', function (
                ?string $monitorSlug = null,
                ?int $checkInMargin = null,
                ?int $maxRuntime = null,
                bool $updateMonitorConfig = true
            ) {
                return $this;
            });
        }
    }
```
to the Console Kernel and to run it at the beggining of the `schedule()` method: `$this->addSentryMonitorMacro();`

**Comment by cleptric** [comment]
@stayallive can you take a look please?

**Comment by rzv-me** [comment]
I submitted a PR for this, the better solution I think is to do it in the `register()` method of `ConsoleIntegration`

**Comment by cleptric** [comment]
Fixed in [4.1.1](https://github.com/getsentry/sentry-laravel/releases/tag/4.1.1)
